js/app.js

```diff
@@ -61,3 +65,61 @@ bookList.addEventListener('click', (e) => {
     bookFun.removeBook(targetEl);
   }
 });
+
+// Date and time
+const clockElement = document.getElementById('clock');
+
+function clock() {
+  clockElement.textContent = new Date().toString();
+}
+
+setInterval(clock, 1000);
+
+// nav bar
+/* eslint-disable no-unused-vars */
+const displaySection = function displaySection(section) {
+  const sectionP = document.getElementById('description');
+  const heading = document.getElementById('title');
+  const sectionList = document.getElementById('list');
+  const sectionForm = document.getElementById('form');
+  const sectionContact = document.getElementById('contact');
+
+  switch (section) {
+    case 'list':
+      heading.innerHTML = 'All Awesome Books';
+      sectionList.style.display = 'block';
+      sectionForm.style.display = 'none';
+      sectionContact.style.display = 'none';
+      sectionP.style.display = 'none';
+      break;
+
+    case 'form':
+      heading.innerHTML = 'Add a New Book';
+      sectionList.style.display = 'none';
+      sectionForm.style.display = 'block';
+      sectionContact.style.display = 'none';
+      sectionP.style.display = 'none';
+      break;
+
+    case 'contact':
+      heading.innerHTML = 'Contact Information';
+      sectionList.style.display = 'none';
+      sectionForm.style.display = 'none';
+      sectionContact.style.display = 'block';
```

Comment:
- [ OPTIONAL ] Why don't you use the add and remove CSS classes to display or hide the content by using the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) `add` and `remove` methods  instead of using this whole lot of code.